### PR TITLE
rdesktop: update to 1.8.6

### PR DESCRIPTION
**Pull request by chrstphrchvz**
###### Type(s)

- [x] bugfix
- [ ] enhancement
- [x] security fix

###### Tested on

macOS 10.14.5 18F132
Xcode 10.2.1 10E1001

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: net/rdesktop/Portfile [hunk 1]
```diff
@@ -33,6 +33,9 @@ depends_lib         port:xorg-libX11 \
                     port:libao \
                     port:libsamplerate
 
+use_autoreconf	yes
+autoreconf.args	-fv
+
```
Comment (chrstphrchvz) [inline net/rdesktop/Portfile]: These lines used to be present in the portfile up until 1089d45425. I found that restoring these prevents an error when attempting to compile which wasn't present with 1.8.4. Example log from earlier Azure build: https://paste.z0k.xyz/de6120fab3b9

```
configure: WARNING: pkg-config not available, cannot check for ao
configure: WARNING: pkg-config not available, cannot check for alsa
configure: WARNING: pkg-config not available, cannot check for samplerate
configure: error: Selected sound system is not available.
```

I'm not sure if this is the right solution for this, so I would appreciate if someone could confirm.

Comment (cjones051073) [reply net/rdesktop/Portfile]: Its a bit odd that it seemingly cannot find `pkg-config` as it is listed as a build-dep... Perhaps the reconf is needed to fix a cached previous incorrect setting...

**Comment by cjones051073** [comment]
@markemer Why did you label this as WIP ?

**Comment by chrstphrchvz** [comment]
> Why did you label this as WIP ?

This was WIP when I opened it. I didn't know how quickly I was going to figure out a solution for the `pkg-config` error.

**Comment by chrstphrchvz** [comment]
> This was WIP when I opened it…

The label can be removed now.

**Comment by cjones051073** [comment]
> > Why did you label this as WIP ?
> 
> This was WIP when I opened it. I didn't know how quickly I was going to figure out a solution for the `pkg-config` error.

Ah, I see now. You only set it as WIP via the PR title. Just for future reference if you wish to make another WIP PR, please (also) use the explicit WIP label, which is what I was looking at.

**Pull request merged**